hail/python/hail/backend/service_backend.py

```diff
@@ -418,34 +418,40 @@ async def _rpc(self,
                     raise
 
             with timings.step("read output"):
```

Comment:
This diff is easier to read with whitespace differences disabled.

Reply:
much